pkg/gardenlet/controller/seed/care/health.go

```diff
@@ -53,52 +54,51 @@ var requiredManagedResourcesSeed = sets.New(
 	vpa.ManagedResourceControlName,
 )
 
-// SeedHealth contains information needed to execute health checks for seed.
-type SeedHealth struct {
-	seed           *gardencorev1beta1.Seed
-	seedClient     client.Client
-	clock          clock.Clock
-	namespace      *string
-	seedIsGarden   bool
-	loggingEnabled bool
+// health contains information needed to execute health checks for a seed.
+type health struct {
```

Comment:
The change took place, but wasn't possible for the `operation/care` package because there is the [CheckClusterNodes](https://github.com/gardener/gardener/blob/078615bdf064840192f97e04d2abd858a263c356/pkg/operation/care/health.go#L541) function for which we have a unit test.